docs/HACKING.md

```diff
@@ -146,68 +60,47 @@ You do not need to set the spec.tls.key, spec.tls.certificate and spec.tls.caCer
 
 Kubernetes TBD+ and OpenShift TBD+ are supported.
 
-- [x] SSL-secured deployment (using Searchguard)
+- [x] SSL-secured deployment (using Opendistro)
 - [x] Insecure deployment (requires different image)
 - [x] Index per tenant
-- [x] Logging to a file or to console
-- [ ] Elasticsearch 6.x support
-- [x] Elasticsearch 5.6.x support
+- [x] Elasticsearch 6.x support
 - [x] Master role
 - [x] Client role
 - [x] Data role
 - [x] Clientdata role
 - [x] Clientdatamaster role
-- [ ] Elasticsearch snapshots
 - [x] Prometheus monitoring
-- [ ] Status monitoring
-- [ ] Rolling restarts
+- [x] Status monitoring
+- [x] Rolling restarts
 
 # Testing
 
 In a real deployment OpenShift monitoring will be installed.  However
 for testing purposes, you should install the monitoring CRDs:
 ```
-[REMOTE_REGISTRY=true] make deploy-setup
+make deploy
 ```
 
-Use `REMOTE_REGISTRY=true make deploy-image` to build the image and copy it
-to the remote registry.
-
 ## E2E Testing
-To run the e2e tests, install the above CRDs and from the repo directory, run:
+To run the e2e tests run:
 ```
-make test-e2e
+make test-e2e-olm
 ```
 This assumes:
 
-* the operator-sdk installed (e.g. `make operator-sdk`)
-* the operator image is built (e.g. `make image`) and available to the OKD cluster
-
-**Note:** It is necessary to set the `IMAGE_ELASTICSEARCH_OPERATOR` environment variable to a valid pull spec in order to run this test against local changes to the `elasticsearch-operator`. For example:
-```
-make deploy-image && \
-IMAGE_ELASTICSEARCH_OPERATOR=quay.io/openshift/origin-elasticsearch-operator:latest make test-e2e
-```
+* the Elasticsearch Operator is already deployed and running
 
 ## Dev Testing
-You should first ensure that you have commands such as `imagebuilder` and `operator-sdk`
-available by using something like `https://github.com/openshift/origin-aggregated-logging/blob/master/hack/sdk_setup.sh`.
+You should first ensure that you have commands such as `podman` and `operator-sdk`
+available.
```

Comment:
We don't require `operator-sdk` installed oin your host anymore, it is provided by the make targets.